Misc/NEWS.d/next/Library/2020-02-25-00-43-22.bpo-39744.hgK689.rst

```diff
@@ -0,0 +1 @@
+Make func:``asyncio.subprocess.Process.communicate`` close subprocess's stdin even when called with input=None
```

Comment:
```suggestion
Make func:`asyncio.subprocess.Process.communicate` close the subprocess's stdin even when called with `input=None`.
```

Reply:
I just changed single quotes to double ones, because the build failed and said I need to use double ones...